packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -904,6 +903,73 @@ def test_prom_metrics_containers_app_bridge(dcos_api_session):
             assert_app_metric_value_for_task(dcos_api_session, node, task_name, metric_name, metric_value)
 
 
+def test_task_prom_metrics_not_filtered(dcos_api_session):
+    """Assert that prometheus app metrics aren't filtered according to adminrouter config.
+
+    This is a regression test protecting a fix for a bug that mistakenly applied filter criteria intended for
+    adminrouter metrics to Prometheus-formatted metrics gathered from tasks.
+
+    """
+    task_name = 'test-task-prom-metrics-not-filtered'
+    metric_name_pfx = 'test_task_prom_metrics_not_filtered'
+    marathon_app = {
+        'id': '/' + task_name,
+        'instances': 1,
+        'cpus': 0.1,
+        'mem': 128,
+        'cmd': '\n'.join([
+            'echo "Creating metrics file..."',
+
+            # Adminrouter metrics with direction="[1-5]xx" tags get dropped.
```

Comment:
these comments make it sound like we're expecting these metrics to get dropped - but they don't because this is an app metric test. I know the docstring for the test explains it better, but to me, these inline comments seemed like they contradicted the docstring; maybe just another line clarifying that if these were adminrouter metrics, they would be filtered out but because this is a test for app metrics, those filters don't apply here?

Reply:
I pushed a commit that adds a comment to the top of this script, clarifying that we expect these metrics to appear because they are task metrics.